src/config/dynamic-theme-fixes.config

```diff
@@ -25530,6 +25556,10 @@ body.skin--responsive,
 #menus-cover-background {
   background-image: linear-gradient(rgb(24, 26, 27) 50%, var(--darkreader-neutral-background) 100%) !important;
 }
+.mwe-math-fallback-image-inline svg g,
```

Comment:
Element does not exist, math element is not inverted, see https://github.com/darkreader/darkreader/issues/12001#issuecomment-1879746162